purchase_order_line_invoicing/wizards/purchase_order_line_invoicing_wizard.py

```diff
@@ -30,8 +30,8 @@ def _prepare_default_line_vals(self, purchase_line):
             'price_subtotal': purchase_line.price_subtotal,
             'qty_received': purchase_line.qty_received,
             'qty_invoiced': purchase_line.qty_invoiced,
-            'invoice_qty':
-                purchase_line.qty_received - purchase_line.qty_invoiced,
+            'invoice_qty': max(
```

Comment:
@benwillig Is this correct as qty_invoiced can be negative in case of in_refund (see https://github.com/odoo/odoo/blob/d8e49d3886ab7c5443b9173bd9b3e45e05b72c6c/addons/purchase/models/purchase.py#L552) ?

Reply:
Indeed the quantity invoiced field can be negative in case of refunds. But I don't think user wants to invoice negative quantities. If we want to tell the user that there was refunds, we should display the quantity invoiced field in the wizard.